packages/vapor/scss/components/table-hoc.scss

```diff
@@ -0,0 +1,31 @@
+.table-predicate-filters + .table-predicate-filters {
+    padding-left: 0;
+    border-left: none;
+}
+
+.table-row-number {
+    :global(.coveo-styleguide .table.table-numbered) {
+        tr:global(.selected) td {
+            &:first-child::before {
+                display: none;
+            }
+
+            &:nth-child(2)::before {
+                @extend %selection-indicator;
+            }
+        }
+    }
+
+    :global(.coveo-styleguide .table) .table-numbered-row:first-child {
+        width: 60px;
+        padding: 0;
+    }
+
+    :global(.coveo-styleguide .table) .table-numbered-row-container {
+        width: 60px;
+        padding: $default-margin;
+        color: var(--medium-blue);
```

Comment:
I see a medium-blue color here :eyes: If it's not difficult, we could replace this deprecated color in this PR. Otherwise, we can also update it in another PR since it's out of scope.

Reply:
Yeah I did not change any css since it's out of scope. We will come for them in due time :eyes: